src/components/__tests__/__snapshots__/tag.test.tsx.snap

```diff
@@ -95,11 +99,17 @@ exports[`Component: Tag snapshot 2`] = `
   font-size: 7px;
   margin-left: 5px;
   height: inherit;
-  padding: 9px 10px;
+  padding: 10px 10px;
   cursor: pointer;
   position: relative;
 }
 
+.emotion-0 i:hover {
```

Comment:
Why are we targetting all `<i>`'s inside this class.
I know that we use `<i>` for icon but someone can create `<i>` in label prop which would also be targetted by this selector.

can we change to 
Tag.styles.ts
```js
export const iconClass = css({
  // icon styles
  ":hover": {
    // icon hover styles
  }
})
```

pebble/src/components/Tag.tsx
```jsx
import { tagStyle, iconClass } from "./styles/Tag.styles";

// in the component
<i className={cx('pi pi-icon-name', iconClass)} />
```